commands.js

```diff
@@ -31,10 +32,11 @@ const addInitOption = (program) => {
 const addCompileOption = (program) => {
   program
     .command('compile')
-    .option('--path [path]', 'Path to contract files', './contracts')
+    .option('-u --url [hostname]', 'Node to connect to', config.localhost)
```

Comment:
rename the option to --network, and the argument passed to network also

Reply:
Network might be used to specify local,ropsten,rinkeby etc, **url** is more suitable?

Reply:
nodeUrl sounds great